vector.go

```diff
@@ -173,7 +183,7 @@ func (v VectorType) Unmarshal(data []byte, value interface{}) error {
 		}
 		return nil
 	}
-	return unmarshalErrorf("can not unmarshal %s into %T. Accepted types: slice, array.", v, value)
+	return unmarshalErrorf("can not unmarshal %s into %T. Accepted types: slice, array, *interface{}.", v, value)
```

Comment:
this should just be `interface{}` or we should make the others be `*slice`, `*array`